community/event-checklist.md

```diff
@@ -0,0 +1,96 @@
+toc=true
+
++++
+
+# 1. Logistics - Before the Workshop
+
+✔️Decide the date and time of the event as per the convenience of speakers and audience. Conducting events during exams or some other important event might not be a good idea in general. Checking academic calendar and other events in KGP is suggested.
+
+✔️Publicize the event at least 2 or 3 days before the workshop. Publicizing earlier will fade away the event info from memory and publicizing late will be difficult for audience to plan.
+
+✔️Share the Post via Facebook, Twitter and Linkedin via KOSS Account.
+
+✔️Speakers should be decided atleast a week before the workshop, giving them enough time to prepare.
+
+✔️It is important to have backup speakers for atleast 2 levels.(A backup speaker for the decided speaker. Another backup speaker for the backup speaker)
+
+✔️Speakers of the event should have enough practical experience with the topic. The experience is necessary so that speaker can convey the content in the simplest way and clear doubts.
+
+✔️It is highly recommended that the Slides for the event are made from scratch by the speaker.  Preparing from scratch will add creativity and newness to the content. New and simpler ways to present a topic might come out.  Also...
+
+​      🗸Feel free to look up at previous slides at [kossiitkgp/slides](https://github.com/kossiitkgp/slides) repo for inspiration
+
+​     🗸Make sure that something important that has been covered in the past doesn't gets removed.
+
+​      🗸Feel free to remove or edit stuff if you ever feel some part of the content is unnecessary and irrelevant, after discussing with team mates. 
+
+✔️Make sure to take feedback on the slides(content) from other members of KOSS regarding the slides. It is suggested to have the discussion over a meet to discuss things like potential queries from audience, thinking of ways to simplify content.
+
+✔️Refer [kossiitkgp/mistakes](https://github.com/kossiitkgp/mistakes) repository to get an idea of what sought of things can go wrong.
+
+✔️It is a good idea to record the event(offline/online) for the sake of posterity.
+
+✔️Make a Github repo for the workshop with name as event name and year which contains - installation instructions, slides, references, recording and others. (For example refer - [1](https://github.com/kossiitkgp/git-and-github-workshop-2020), [2](https://github.com/kossiitkgp/REST-APIs-in-Flask-Workshop-2021), [3](https://github.com/kossiitkgp/Linux-Shell-Vim-Workshop-2021))
+
+## 1.1 Internal
+
+- Keep in touch regularly with other team mates on updates regarding the event. Regular communication on #events channel is suggested.
+- Meet regularly to discuss stuff. It is advisable to send mail invites even for a smaller meet, to avoid procrastination.
```

Comment:
```suggestion
- Meet regularly to discuss stuff. It is advisable to send **_calendar invites_** even for a smaller meet, to avoid procrastination.
```